### PR TITLE
TextInput Component

### PR DESCRIPTION
**Pull request by mshago**
I created a new branch with the Text Input Component

### CODE REVIEW

File: src/components/textInput/TextInput.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  isSensitive = false,
+  style:customStyles,
+  ...rest
+  
+}) => {
+
+  return (
+    <NativeTextInput
+      style={[styles.textInputStyles, customStyles]}
+      placeholder={placeHolder}
```
Comment (lasharela) [inline src/components/textInput/TextInput.js:18]: remove

File: src/components/textInput/TextInput.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import {TextInput as NativeTextInput } from 'react-native';
+import React from 'react';
+import {styles} from './TextInput.styles';
+import {color} from '../../theme/'
+
+
+export const TextInput = ({
+  placeHolder,
```
Comment (lasharela) [inline src/components/textInput/TextInput.js:8]: remote

File: src/components/textInput/TextInput.styles.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import {StyleSheet} from 'react-native';
+import {color, size} from '../../theme'
+/*
+ * Styles used for TextInput component
+ */
+
+export const styles = StyleSheet.create({
+  textInputStyles: {
+    backgroundColor: color.white,//White
```
Comment (lasharela) [inline src/components/textInput/TextInput.styles.js:9]: no extra comments. no pollution in source

**Review by lasharela — APPROVED** [review]

**Review by raynesmike — APPROVED** [review]
I approve this component. Great stuff

**Review by carizzuti — APPROVED** [review]
Component looks good

**Pull request merged**